src/sparseml/modifiers/obcq/utils/models.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+import torch
+from torch.nn import Module
+from transformers import LlamaForCausalLM, OPTForCausalLM
+
+from sparseml.modifiers.obcq.utils.utils import (
+    cache_attention_inputs,
+    execute_offloaded_module,
+)
+
+
+__all__ = ["load_opt_model", "load_llama_model", "opt_forward", "llama_forward"]
+
+
+def load_opt_model(model_path: str) -> torch.nn.Module:
```

Comment:
should probably expand this out to something like this: https://github.com/neuralmagic/sparseml/blob/058aa512c5efa5e85ab0b88cb49bf930aec5f5e4/src/sparseml/transformers/utils/model.py#L40 for standardization with re-loading and export (and also managing sparse state)